antarest/study/storage/rawstudy/raw_study_service.py

```
@@ -45,6 +46,18 @@
 logger = logging.getLogger(__name__)
 
 
+def copy_output_folders(src_output_path: Path, dest_output_path: Path, allowed_outputs: List[str]) -> None:
```

Why not simply using a shutil.copytree ?

Also change the line 58. IMO we should raise an Exception if the name doesn't exist